README.md

```diff
@@ -80,6 +80,9 @@ Once you have your template complete, check out our [health architectures](https
 * [Helper function summary](docs/helper-functions-summary.md)
 * [Web UI summary](docs/web-ui-summary.md)
 
+### Known issues
+ * [The Web UI auto-update issue](docs/web-ui-auto-update-issue.md)
+

```

Comment:
Describe the issue in one sentence: The V1 converter UI gets auto-updated to V2, and the templates are no longer visible in the UI. Follow these (link) steps to resolve the issue